toolkit/src/components/cut/filter-bar/index.ts

```diff
@@ -244,7 +200,9 @@ export default class FilterBarComponent extends Component<ComponentSignature> {
     const config = structuredClone(this.args.config);
 
     // apply the filter from filterChanges
-    if (Object.hasOwn(this.pendingFilterChange || {}, name)) {
```

Comment:
TS was complaining about this as it was added in ES2022. I just updated it to used the prototypes `hasOwnProperty` as that will suffice for this check.